### PR TITLE
[SCB-1645] update license

### PR DESCRIPTION
**Pull request by kakulisen**

### CODE REVIEW

File: toolkit-distribution/src/release/LICENSE [hunk 1]
```diff
@@ -319,10 +319,13 @@ This product bundles the following libraries which are licensed
 under the Apache License v2.
 For details, see their respective project links.
 
+* airline (io.airlift:airline:0.7 - https://github.com/airlift/airline)
```
Comment (chanjarster) [inline toolkit-distribution/src/release/LICENSE]: It's already declared on line 401: https://github.com/apache/servicecomb-toolkit/blob/master/toolkit-distribution/src/release/LICENSE#L410

File: toolkit-distribution/src/release/LICENSE [hunk 2]
```diff
@@ -454,6 +456,7 @@ For details, see their respective project links.
 * swagger-parser (io.swagger:swagger-parser:1.0.44 - http://nexus.sonatype.org/oss-repository-hosting.html/swagger-parser-project/modules/swagger-parser)
 * swagger-parser (org.openapitools.swagger.parser:swagger-parser-core:2.0.14-OpenAPITools.org-1 - http://nexus.sonatype.org/oss-repository-hosting.html/swagger-parser-project/modules/swagger-parser-core)
 * swagger-parser (org.openapitools.swagger.parser:swagger-parser:2.0.14-OpenAPITools.org-1 - http://nexus.sonatype.org/oss-repository-hosting.html/swagger-parser-project/modules/swagger-parser)
+* swagger-parser (org.openapitools.swagger.parser:swagger-parser-core:2.0.14-OpenAPITools.org-1 - http://nexus.sonatype.org/oss-repository-hosting.html/swagger-parser-project/modules/swagger-parser-core)
```
Comment (chanjarster) [inline toolkit-distribution/src/release/LICENSE]: It's already declared in line 455: https://github.com/apache/servicecomb-toolkit/blob/master/toolkit-distribution/src/release/LICENSE#L455

File: toolkit-distribution/src/release/LICENSE [hunk 3]
```diff
@@ -465,6 +468,8 @@ For details, see their respective project links.
 * tomcat-embed-el (org.apache.tomcat.embed:tomcat-embed-el:9.0.17 - https://tomcat.apache.org/)
 * tomcat-embed-websocket (org.apache.tomcat.embed:tomcat-embed-websocket:9.0.17 - https://tomcat.apache.org/)
 * unbescape (org.unbescape:unbescape:1.1.6.RELEASE - http://www.unbescape.org)
+* Woodstox (com.fasterxml.woodstox:woodstox-core:5.0.3 - https://github.com/FasterXML/woodstox)
```
Comment (chanjarster) [inline toolkit-distribution/src/release/LICENSE]: It's already declared in line 409: https://github.com/apache/servicecomb-toolkit/blob/master/toolkit-distribution/src/release/LICENSE#L409

File: toolkit-distribution/src/release/LICENSE [hunk 4]
```diff
@@ -533,3 +538,24 @@ For details, see http://asm.objectweb.org/asm/
...
+===========================================================================
+This product bundles jmustache which is licensed under The (New) BSD License.
+For details, see http://github.com/samskivert/jmustache
+You can find a copy of the License at licenses/LICENSE-jmustache
+
+* jmustache (com.samskivert:jmustache:1.14 - http://github.com/samskivert/jmustache)
+
+===========================================================================
+This product bundles JSch which is licensed under Revised BSD.
+For details, see http://www.jcraft.com/jsch/
```
Comment (chanjarster) [inline toolkit-distribution/src/release/LICENSE]: should be http://www.jcraft.com/jsch/LICENSE.txt

File: toolkit-distribution/src/release/LICENSE [hunk 5]
```diff
@@ -533,3 +538,24 @@ For details, see http://asm.objectweb.org/asm/
...
+You can find a copy of the License at licenses/LICENSE-jsch
+
+* JSch (com.jcraft:jsch:0.1.54 - http://www.jcraft.com/jsch/)
+
+===========================================================================
+This product bundles JZlib which is licensed under Revised BSD.
+For details, see http://www.jcraft.com/jzlib/
```
Comment (chanjarster) [inline toolkit-distribution/src/release/LICENSE]: should be http://www.jcraft.com/jzlib/LICENSE.txt

File: toolkit-distribution/src/release/licenses/LICENSE-jsch [hunk 6]
```diff
@@ -0,0 +1,30 @@
+JSch 0.0.* was released under the GNU LGPL license.  Later, we have switched
```
Comment (chanjarster) [inline toolkit-distribution/src/release/licenses/LICENSE-jsch]: @WillemJiang I'm not sure this is ok

File: toolkit-distribution/src/release/licenses/LICENSE-jzlib [hunk 7]
```diff
@@ -0,0 +1,29 @@
+JZlib 0.0.* were released under the GNU LGPL license.  Later, we have switched
```
Comment (chanjarster) [inline toolkit-distribution/src/release/licenses/LICENSE-jzlib]: I'm not sure this is OK

File: toolkit-distribution/src/release/licenses/LICENSE-jsch (hunk 6 above)
Comment (WillemJiang) [reply toolkit-distribution/src/release/licenses/LICENSE-jsch]: It's fine once the License is BSD, I just checked the we are using JSch 0.1.x which is BSD license.

File: toolkit-distribution/src/release/LICENSE (hunk 3 above)
Comment (WillemJiang) [reply toolkit-distribution/src/release/LICENSE]: We should remove the duplicated line.

File: toolkit-distribution/src/release/licenses/LICENSE-jzlib (hunk 7 above)
Comment (WillemJiang) [reply toolkit-distribution/src/release/licenses/LICENSE-jzlib]: It's fine.

**Comment by kakulisen** [comment]
Thanks for the review, I have fixed the duplicated line issue and correct the URL

File: toolkit-distribution/src/release/LICENSE (hunk 4 above)
Comment (kakulisen) [reply toolkit-distribution/src/release/LICENSE]: Ok, I have removed it in the new commit

File: toolkit-distribution/src/release/LICENSE (hunk 1 above)
Comment (kakulisen) [reply toolkit-distribution/src/release/LICENSE]: Ok, I have removed it in the new commit

File: toolkit-distribution/src/release/LICENSE (hunk 2 above)
Comment (kakulisen) [reply toolkit-distribution/src/release/LICENSE]: Ok, I have removed it in the new commit

File: toolkit-distribution/src/release/LICENSE (hunk 3 above)
Comment (kakulisen) [reply toolkit-distribution/src/release/LICENSE]: Ok, I have removed it in the new commit

File: toolkit-distribution/src/release/LICENSE (hunk 5 above)
Comment (kakulisen) [reply toolkit-distribution/src/release/LICENSE]: I have correct the URL in the new commit

File: toolkit-distribution/src/release/LICENSE (hunk 4 above)
Comment (kakulisen) [reply toolkit-distribution/src/release/LICENSE]: I have correct the URL in the new commit

**Comment by chanjarster** [comment]
Seems issue fixed, thanks @kakulisen

**Comment by kakulisen** [comment]
@chanjarster Thanks for your method, it is very useful
@WillemJiang  please check it again, thanks

**Comment by chanjarster** [comment]
LGTM

**Pull request merged**